owncloudApp/src/main/java/com/owncloud/android/presentation/viewmodels/security/PatternViewModel.kt

```diff
@@ -20,12 +20,19 @@
 
 package com.owncloud.android.presentation.viewmodels.security
 
+import android.content.SharedPreferences
+import android.os.Build
 import androidx.lifecycle.ViewModel
 import com.owncloud.android.data.preferences.datasources.SharedPreferencesProvider
+import com.owncloud.android.db.PreferenceManager
+import com.owncloud.android.presentation.ui.security.BiometricActivity
+import com.owncloud.android.presentation.ui.security.BiometricManager
 import com.owncloud.android.presentation.ui.security.PatternActivity
+import com.owncloud.android.providers.ContextProvider
 
 class PatternViewModel(
-    private val preferencesProvider: SharedPreferencesProvider
+    private val preferencesProvider: SharedPreferencesProvider,
+    private val contextProvider: ContextProvider,
```

Comment:
This won't be necessary! 😀  Check next comment

Reply:
Done